api/api.go

```diff
@@ -162,6 +162,10 @@ type QueryOptions struct {
 	// Filter requests filtering data prior to it being returned. The string
 	// is a go-bexpr compatible expression.
 	Filter string
+
+	// Prune is an optional context pass to allow forceleave to
+	// remove all knowledge of an agent
+	Prune bool
```

Comment:
This does not belong on QueryOptions. This struct is for things that apply to all\* endpoints.